TYPO3.Neos/Resources/Public/JavaScript/Shared/KeyboardShortcutsDialog.html

```diff
@@ -0,0 +1,110 @@
+<div class="neos-modal neos-modal-wide neos-show-shortcuts">
+    <div class="neos-modal-header">
+        <button class="neos-close neos-button" title="Close" {{action "hide" target="view"}}></button>
+        <div class="neos-header">{{translate id="keyboard.shortcuts.header" fallback="Keyboard shortcuts"}}</div>
+    </div>
+    <div class="neos-modal-body">
+        <div class="neos-column-body">
+            <div class="neos-subheader">
+                {{translate id="keyboard.shortcuts.general" fallback="General"}}
+            </div>
+            <ul class="neos-shortcut-list neos-clearfix">
+                <li class="neos-shortcut">
+                    <div class="neos-shortcut-body">
+                        <div class="neos-shortcut-icon">
+                            <i class="icon-resize-full"></i>
+                        </div>
+                        <div class="neos-shortcut-label">{{translate id="keyboard.shortcuts.toggleFullScreen" fallback="Toggle fullscreen mode"}}</div>
+                        <div class="neos-shortcut-keys">{{modKey}} + h</div>
+                    </div>
+                </li>
+                <li class="neos-shortcut">
+                    <div class="neos-shortcut-body">
+                        <div class="neos-shortcut-icon">
+                            <i class="icon-remove"></i>
+                        </div>
+                        <div class="neos-shortcut-label">{{translate id="keyboard.shortcuts.closeDialog" fallback="Close any dialog box"}}</div>
+                        <div class="neos-shortcut-keys">esc</div>
```

Comment:
Yea, but i think @aertmann point is to wrap it into the translation ViewHelper and add it to the translation files so it can be translated on Crowdin if necessary